LibOS/shim/include/shim_defs.h

```diff
@@ -13,6 +13,11 @@
 #define DEFAULT_BRK_MAX_SIZE        (256 * 1024)        /* 256KB */
 #define DEFAULT_SYS_STACK_SIZE      (256 * 1024)        /* 256KB */
 
+#define DEFAULT_AUXV_NUM        (8)
+#define AT_RANDOM_SIZE      (16)
```

Comment:
Can you add a comment what this macro is for?

Reply:
Done